requirements-dev.txt

```diff
@@ -2,3 +2,4 @@
 pandas
```

Comment:
Is `pandas` a dev requirement? It looks like it's needed for `predict` which will be used in production?